app/src/main/java/com/vmware/nimbus/api/APIService.java

```diff
@@ -58,6 +59,7 @@ public void onErrorResponse(VolleyError error) {
                         Log.d("volley", "Error: " + error.getMessage());
                         error.printStackTrace();
                         callback.onFailure(false);
+//                        toastMsg("API SERVICE - log in failed", c);
```

Comment:
Can we remove instead of comment? If we want it back we can just go back in the git history.